tests/scripts/build_dgl.sh

```diff
@@ -15,7 +15,10 @@ if [ "$1" != "cugraph" ]; then
     # We do not build pytorch for cugraph because currently building
     # pytorch against all the supported cugraph versions is not supported
     # See issue: https://github.com/rapidsai/cudf/issues/8510
-    CMAKE_VARS="$CMAKE_VARS -DBUILD_TORCH=ON -DBUILD_SPARSE=ON -DTORCH_PYTHON_INTERPS=/opt/conda/envs/pytorch-ci/bin/python"
+    CMAKE_VARS="$CMAKE_VARS -DBUILD_TORCH=ON -DTORCH_PYTHON_INTERPS=/opt/conda/envs/pytorch-ci/bin/python"
+else
+    # Disable sparse build as cugraph docker image lacks cuDNN.
+    CMAKE_VARS="$CMAKE_VARS -DBUILD_SPARSE=OFF"
```

Comment:
Thanks for fixing this.
Just curious, why doesn't cugraph docker image include cuDNN?

Reply:
I don't know. For now `cugraph CI` is running on different image from other DGL CI tests: `rapidsai/cugraph_nightly_torch-cuda:11.5-base-ubuntu18.04-py3.9-pytorch1.12.0-rapids22.12`.

Reply:
@VibhuJawa Hi, we're trying to enable `DGL SPARSE` build in default. Do we need to enable it for cugraph CI? For now, seems the docker image used for cugraph CI does not include `cuDNN`. Could you help confirm this?

Reply:
Let me check and confirm on this.  Also , CC: @tingyu66  from who might also have some input here.

Reply:
It is because the aggregation functions (for GraphSAGE, GAT and RGCN) in `pylibcugraphops` simply do not use cudnn. Also to my knowledge, cugraph does not need cudnn either.

Reply:
thanks for your inputs. then let's do not enable sparse build for cugraph CI for now.